### ISSUE TITLE
broken on meteor 1.6 beta

### ISSUE THREAD

**Issue by alchra** [tozd/docker-meteor#5]
Looks like there are missing dependencies in the meteor version 1.6 beta
Got following error:

```
docker build .
...
Error: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by /.meteor/packages/meteor-tool/.1.6.0-beta.4.4se276++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/fibers/bin/linux-x64-57/fibers.node)
    at Object.Module._extensions..node (module.js:598:18)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/.meteor/packages/meteor-tool/.1.6.0-beta.4.4se276++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/fibers/fibers.js:24:38)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
The command '/bin/sh -c export METEOR_ALLOW_SUPERUSER=true &&  rm -rf /source/.meteor/local /source/node_modules &&  if [ -x /source/docker-source.sh ]; then /source/docker-source.sh; fi &&  cp -a /source /build &&  rm -rf /source &&  cd /build &&  meteor list &&  if [ -f package.json ]; then meteor npm install --production; fi &&  meteor build --headless . &&  cd / &&  tar xf /build/build.tar.gz &&  rm -rf /build' returned a non-zero code: 1

```

**Comment by mitar** [comment]
Can you try using `FROM tozd/meteor:ubuntu-xenial` and report if it solves your problem?

**Comment by alchra** [comment]
that was quick :-)
yes indeed it solved it but here is the next one

```
docker build .
...
npm ERR! code ENOGIT
npm ERR! No git binary found in $PATH
npm ERR! 
npm ERR! Failed using git.
npm ERR! Please check if you have git installed and in your PATH.

npm ERR! A complete log of this run can be found in:
npm ERR!     /.meteor/packages/meteor-tool/.1.6.0-beta.4.1esiq4y++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/.npm/_logs/2017-07-24T13_25_50_851Z-debug.log
The command '/bin/sh -c export METEOR_ALLOW_SUPERUSER=true &&  rm -rf /source/.meteor/local /source/node_modules &&  if [ -x /source/docker-source.sh ]; then /source/docker-source.sh; fi &&  cp -a /source /build &&  rm -rf /source &&  cd /build &&  meteor list &&  if [ -f package.json ]; then meteor npm install --production; fi &&  meteor build --headless . &&  cd / &&  tar xf /build/build.tar.gz &&  rm -rf /build' returned a non-zero code: 1
```

**Comment by mitar** [comment]
Who triggers this last error? Missing `git`? Is this a package which requires it?

**Comment by mitar** [comment]
Because if you require `git` for some of your dependencies, then you should probably [install it yourself](https://github.com/peer/mind/blob/master/docker-source.sh#L4).

But I just added it to this image, because it is probably a common dependency. See [here](https://hub.docker.com/r/tozd/meteor/builds/), and when the images get rebuild, please test and report here.

**Comment by alchra** [comment]
Great, now it works and thank you for the hint with the docker-source.sh file. Maybe worth mentioning in the readme ;-)
And Git is not a dependency of my packages.

**Comment by mitar** [comment]
Great!

Documented: e972d03c8f3576abeb9e71e5ffa50e9ed53e20a5

PRs are welcome. :-)

**Comment by mitar** [comment]
BTW, this was fixed in Meteor 1.6 beta 5 days ago: https://github.com/meteor/meteor/pull/8728#issuecomment-316830755

So now Meteor 1.6 should also work with `tozd/meteor:ubuntu-trusty`.
